hypothesis-python/src/hypothesis/strategies/_internal/core.py

```diff
@@ -529,6 +529,7 @@ def characters(
     min_codepoint: Optional[int] = None,
     max_codepoint: Optional[int] = None,
     whitelist_characters: Optional[Sequence[str]] = None,
+    codec: Optional[str] = None,
```

Comment:
Would literals be fine here?

```suggestion
    codec: Optional[Literal["ascii", "utf8"]] = None,
```